stac_fastapi/elasticsearch/tests/api/test_api.py

```diff
@@ -31,17 +33,20 @@
 }
 
 
+@pytest.mark.asyncio
```

Comment:
You can mark the entire test module as async, e.g. https://github.com/stac-utils/stac-asset/blob/a11b4d217fc6c6479b3b9d9282f668ac9ded432c/tests/test_filesystem_client.py#L7